src/java.base/share/classes/java/lang/classfile/Attributes.java

```diff
@@ -170,7 +174,7 @@ public class Attributes {
     /** RuntimeInvisibleTypeAnnotations */
```

Comment:
this comment should probably be: RuntimeInvisibleParameterAnnotations

Reply:
Fixed, thanks.